manage-operational-layers/src/main/java/com/esri/arcgismaps/sample/manageoperationallayers/screens/MainScreen.kt

```diff
@@ -23,28 +23,32 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import com.esri.arcgismaps.sample.manageoperationallayers.components.ComposeMapView
+import androidx.compose.ui.platform.LocalContext
+import com.arcgismaps.toolkit.geocompose.MapView
+import com.arcgismaps.toolkit.geocompose.MapViewpointOperation
 import com.esri.arcgismaps.sample.manageoperationallayers.components.MapViewModel
 import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
 
 /**
  * Main screen layout for the sample app
  */
 @Composable
-fun MainScreen(sampleName: String, application: Application) {
+fun MainScreen(sampleName: String) {
     // create a ViewModel to handle MapView interactions
-    val mapViewModel = MapViewModel(application)
+    val mapViewModel = MapViewModel(LocalContext.current.applicationContext as Application)
+    // get the instance of the MapView state
+    val mapViewState = mapViewModel.mapViewState
 
     Scaffold(
         topBar = { SampleTopAppBar(title = sampleName) },
         content = {
             Column(
                 modifier = Modifier.fillMaxSize().padding(it)
             ) {
-                // composable function that wraps the MapView
-                ComposeMapView(
+                MapView(
                     modifier = Modifier.fillMaxSize().weight(1f),
-                    mapViewModel = mapViewModel,
+                    arcGISMap = mapViewState.arcGISMap,
```

Comment:
Could you remove the `MapViewState` from the ViewModel? Instead expose `ArcGIS` as a property of the ViewModel. (Similar to [these](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/152/files#diff-57989e124f58d7fb5854e68cb7751c4f8288d99732728128177f8803e8680a01) changes)